### ISSUE TITLE
Checksum assert

### ISSUE THREAD

**Issue by casey** [cberner/redb#359]
I just pushed a branch that updates redb to 0.6.0, and I'm hitting an assert on CI that seems to be checking the checksum type:

```
thread 'main' panicked at 'assertion failed: `(left == right)`
  left: `Zero`,
 right: `XXH3_128`', /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/redb-0.6.0/src/tree_store/page_store/page_manager.rs:883:9
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

https://github.com/casey/ord/runs/8294934765?check_suite_focus=true#step:8:523

**Comment by cberner** [comment]
Oops, I even had a TODO about this, but didn't realize it completely broke `open()` at the time =P

**Comment by cberner** [comment]
Ok, I released 0.6.1 with the fix. Give that a try :)

**Comment by casey** [comment]
Nice, that fixed it
